### PR TITLE
feat

### PR DESCRIPTION
**Pull request by theVivek-kumar**
create my cart page

### CODE REVIEW

**Review by VLeads — APPROVED** [review]

**Review by VLeads — APPROVED** [review]
Nothing to add. Good to go !

File: src/context/wishlistContext.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { createContext, useContext, useReducer, useEffect, useState } from "react";
+const WishListContext = createContext();
+
+
```
Comment (Adarsh-Patel-Dev) [inline src/context/wishlistContext.jsx:5]: remove extra spaces here

File: src/context/wishlistContext.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        headers: {
+            authorization: localStorage.getItem('token')
+        },
+        data: {
+            product: product
+        }
+    });
+       setwishList(response.data.wishlist)
+       console.log("add into wish");
+       console.log(response.data.wishlist)
```
Comment (Adarsh-Patel-Dev) [inline src/context/wishlistContext.jsx:40]: remove console.log

File: src/components/cartPage/cartPage.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import { useContext, useEffect } from "react";
+import axios from "axios";
+import { CardHorizontal } from "../Card/cardHorizontal";
+import { WishListContext } from "../../context/wishlistContext";
+import { Cartprice} from "../cartprice";
+
+const CartPage = () => {
+    
+    const { cart, setCart, addToCart, changeCartQty, removeFromCart } = useContext(CartContext);
+    const { addToWishList,setwishList } = useContext(WishListContext);
```
Comment (Adarsh-Patel-Dev) [inline src/components/cartPage/cartPage.jsx:13]: create and use custom hook for context

File: src/components/cartPage/cartPage.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                <div className="cart-section-card">
+                    {
+                        cart.map(item => (< CardHorizontal product={item} removeFromCart={() =>removeFromCart(item._id,setCart) }
+                            addToWishList={() => addToWishList (item , setwishList) } />))}
+                    
+
+                </div>
+              
+               
+
```
Comment (Adarsh-Patel-Dev) [inline src/components/cartPage/cartPage.jsx:53]: remove spaces here as well

File: src/components/navbar/Navbar.js [hunk 5]
```diff
@@ -15,13 +15,14 @@ const Navbar = () => {
       </div>
       
       <div className='navbar-searchbox-container'>
-          <input className='search-input' placeholder=' Search for procuct,and more...'  ></input> <div className='search-icon'><BiSearch className='search-icon' />
+          <input className='search-input' placeholder='    Search for procuct,and more...'  ></input> <div className='search-icon'><BiSearch className='search-icon' />
           </div>
         
       </div>
       <div className='navbar-icons'>
         {/* <NavLink to = "/cart"><BsCartCheck className='icons' /></NavLink> */}
```
Comment (Adarsh-Patel-Dev) [inline src/components/navbar/Navbar.js:23]: remove comments

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Pull request merged**